src/app/families/families.component.ts

```diff
@@ -705,6 +717,7 @@ export async function saveFamiliesToExcel(context: Context, gs: GridSettings<Fam
             else
                 return p.value.replace(/\D/g, '');
         }
+        addColumn("X" + use.language.email, f.email.value, 's');
```

Comment:
You don't need this line - all columns of the entity are included by default on the export to excel

Reply:
so why are the other similar lines there?

Reply:
These are special fields that on top of having their normal values - we also needed something else - like showing the phones with the original values - or without hiphens